modules/integration_azure-firewall/conf/02-throughput.yaml

```diff
@@ -0,0 +1,25 @@
+## Example
+module: Azure firewall
+name: throughput
+
+transformation: true
+aggregation: true
+
+filtering: "filter('resource_type', 'Microsoft.Network/azureFirewalls') and filter('primary_aggregation_type', 'true')"
+
+tip: "Firewall will autoscale at 2.5Gbps but the maximum throughput is 30Gbps"
+
+signals:
+  throughput:
+    metric: "Throughput"
+  signal:
+    formula: "throughput.scale(0.000000953674316)"
+rules:
+  critical:
+    threshold: 25600
+    comparator: ">"
+    lasting_duration: "900s"
+  major:
+    threshold: 2560
```

Comment:
What's the point of triggering an alert on this threshold ? 
https://docs.microsoft.com/en-us/azure/firewall/firewall-faq#how-can-i-increase-my-firewall-throughput

Reply:
I think we should be alerted when the firewall need to scale up. It may be normal or not. May be a major alert is not the best level ?

Reply:
My 2 cents:
* critical: 29 Gbps
* Major: 27 Gbps
* Minor: 25Gbps
* Warning: 3Gbps 

Firewall scaling do not need any run operation